cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -31,8 +31,10 @@ import (
 	"github.com/pkg/errors"
 	autoscalingv1 "k8s.io/api/autoscaling/v1"
 	apiv1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
This seems like a duplicate of the import above

Reply:
Fixed thanks!